### PR TITLE
LPS-159790 [Raylife Ap] Creating PRODUCT PERFORMANCE Basic Structure

### PR DESCRIPTION
**Pull request by JoMaAlves**

### CODE REVIEW

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/dashboard/pages/ProductPerformance.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+					<ClaySelect
+						className="product-performance-select"
+						onChange={({target}) => {
+							setTimePeriod(target.value);
+						}}
+						sizing="sm"
+						value={timePeriod}
+					>
+						{TIME_PERIODS.map((item, index) => (
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/dashboard/pages/ProductPerformance.tsx:82]: Please, change it.
```suggestion
						{TIME_PERIODS.map((time_period, index) => (
```

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions. :fire:

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
ci:relevant

**Comment by JoMaAlves** [comment]
ci:test:relevant

**Review by silasbsi — APPROVED** [review]

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @JoMaAlves

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**